R/epiprocess.R

```diff
@@ -12,4 +12,7 @@
 #' @importFrom rlang %||%
 #' @name epiprocess
 "_PACKAGE"
-utils::globalVariables(c(".x", ".group_key", ".ref_time_value"))
+utils::globalVariables(c(
+  ".x", ".group_key", ".ref_time_value", "resid",
+  "fitted", ".response", "geo_value", "time_value"
```

Comment:
We had some NSE column names from dplyr handling causing "not defined as global variable" warnings. In other places in this repo we use `.data$...` to use them, and sometimes we don't. I'm not sure if we have a standard there, so I've added them here (and some others whose warnings can't be suppressed in another way)

Reply:
Sounds good for a quick fix!  Not sure we need to settle on a standard right now.  Brainstorming in #479.